libc/stdlib/itoa.c

```diff
@@ -0,0 +1,34 @@
+#include <string.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+
+char get_ascii(int number){
```

Comment:
Can you elaborate on that function
I understand that number of value 0-9 will be converted into '0'-'9' and number of value 10 means 'A' and so on.
But what about small case letters? They start from value 97 and there is a gap between big case letters and small case letters.
![image](https://user-images.githubusercontent.com/58564314/199632156-1f5f75af-444e-4546-a967-2c8245e3e4f2.png)
If on the other hand we consider those signs (marked in yellow) what about signs with ASCI value 58-64.
From what I can see this function is unable to return them.
![image](https://user-images.githubusercontent.com/58564314/199632386-e96f08b7-4318-4a2b-90fd-c8e72147e4f7.png)